pkgs/applications/networking/StevenBlack-hosts/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenvNoCC, fetchFromGitHub, python3, lib
+, flags ? {} ,whitelist ? ""}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "StevenBlack-hosts";
+  version = "2.5.52";
+  src = fetchFromGitHub {
+    owner = "StevenBlack";
+    repo = "hosts";
+    rev = version;
+    sha256 ="zQMzXci1/21PCvi8AGqyDataQl/kQJJ+jszxXd2XMQc=";
```

Comment:
I'd suggest using the base32 hash format which is more prevalent in nixpkgs and used by tools such as `nix-prefetch-url`:
```
nix-hash --type sha256 --to-base32 "zQMzXci1/21PCvi8AGqyDataQl/kQJJ+jszxXd2XMQc="
```